### ISSUE TITLE
Request to be able to add users to groups, and direct server-specific alerts to specific groups

### ISSUE THREAD

**Issue by mikeizzy** [pganalyze/collector#305]
It appears all alerts for a server are only configurable to either be sent (or not) to all users individually.
https://pganalyze.com/docs/checks

We have a multi-tenant model where we wish to alert the specific owners (which would be mapped to a group) about specific issues on specific servers (in the one organisation).

**Comment by mikeizzy** [comment]
We are verifying whether _custom roles_ is able to segregate alerts as described above. If this is the case, this FR can be deferred.

**Comment by mikeizzy** [comment]
_Custom roles_ appears to be able to serve this function.

**Comment by lfittl** [comment]
@mikeizzy Thats right, thats the idea. We also have some upcoming improvements for alerts that will make this more straightforward and easier to customize. You may have noticed already, but right now the roles work for setting email-based alerts using the alert policies, but not for Slack/PagerDuty.

Also, for reference, for feature requests its usually best to file a support ticket, then we can add it to our internal tracking for feature requests (we use the collector repository mainly for the collector component itself).
